app/common/directives/color-picker.js

```diff
@@ -29,7 +29,7 @@ module.exports = [
         return {
             restrict: 'E',
             replace: true,
-            templateUrl: 'templates/common/color-picker/color-picker.html',
+            templateUrl: 'templates/common/directives/color-picker.html',
```

Comment:
Can you just push this fix directly onto develop?
(aside: I picked up a bunch of errors like this in the webpack branch because using require() means the build checks if paths actually exist)